automatic/iTunes/iTunes.nuspec

```diff
@@ -1,28 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
-<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
-    <id>{{PackageName}}</id>
-    <version>{{PackageVersion}}</version>
-    <title>iTunes</title>
+    <id>iTunes</id>
```

Comment:
true, it should.
But until https://github.com/chocolatey/choco/issues/1040 is fixed, it shouldn't be changed for existing packages.